### PR TITLE
Inform governance-approved agenda

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

**Review by lemonholic — APPROVED** [review]

File: node/src/node.rs [hunk 1]
```diff
@@ -264,6 +266,41 @@ impl<N: GossipNetwork, S: Storage, R: RawRepository> SimperbyApi for Node<N, S,
...
         futures::try_join!(t1, t2, t3)?;
+
+        let governance_set = self
+            .last_reserved_state
+            .get_governance_set()
+            .unwrap()
+            .into_iter()
+            .collect::<HashMap<_, _>>();
+        let governance_state = self.governance.read().await?;
+        let total_voted_powers: Vec<(Hash256, VotingPower)> = governance_state
```
Comment (TomTaehoonKim) [inline node/src/node.rs:277]:
```suggestion
        let total_voted_power: Vec<(Hash256, VotingPower)> = governance_state
```

File: node/src/node.rs [hunk 2]
```diff
@@ -264,6 +266,41 @@ impl<N: GossipNetwork, S: Storage, R: RawRepository> SimperbyApi for Node<N, S,
...
+                    votes
+                        .keys()
+                        .map(|voter| governance_set.get(voter).unwrap())
+                        .sum(),
+                )
+            })
+            .collect();
+        let total_voting_power = governance_set.values().sum::<VotingPower>();
+        for (agenda, voted_power) in total_voted_powers {
+            if voted_power * 2 > total_voting_power {
```
Comment (TomTaehoonKim) [inline node/src/node.rs:292]: agendas are approved on majority right?

Comment (junha1) [reply node/src/node.rs:292]: yes

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Review by YoonhoKim0527 — APPROVED** [review]
LGTM

**Pull request merged**